src/test/java/org/phyloref/jphyloref/helpers/OWLHelperTest.java

```diff
@@ -0,0 +1,114 @@
+      // Set up a set of axioms we will use to run these tests.
+      ontologyManager = OWLManager.createOWLOntologyManager();
+      OWLDataFactory df = ontologyManager.getOWLDataFactory();
+
+      // A property for RDFS_LABEL.
+      OWLAnnotationProperty RDFSLabelProperty =
+          df.getOWLAnnotationProperty(OWLRDFVocabulary.RDFS_LABEL.getIRI());
+
+      // Set up some labels on a named individual.
+      List<OWLAxiom> axioms = new ArrayList<>();
+      IRI phyloref1IRI = IRI.create("http://example.org/phyloref1");
+
+      axioms.add(
+          df.getOWLAnnotationAssertionAxiom(
+              RDFSLabelProperty, phyloref1IRI, df.getOWLLiteral("Label without a language", "")));
+      axioms.add(
+          df.getOWLAnnotationAssertionAxiom(
+              RDFSLabelProperty, phyloref1IRI, df.getOWLLiteral("Label in English", "en")));
+      axioms.add(
+          df.getOWLAnnotationAssertionAxiom(
+              RDFSLabelProperty, phyloref1IRI, df.getOWLLiteral("Etikett auf Englisch", "de")));
+      axioms.add(
+          df.getOWLAnnotationAssertionAxiom(
+              RDFSLabelProperty, phyloref1IRI, df.getOWLLiteral("अंग्रेजी में लेबल", "hi")));
+
+      // Set up the test ontology.
+      testOntology = ontologyManager.createOntology(new HashSet<>(axioms));
+    }
+
+    @Test
+    @DisplayName("read labels in English")
+    void canReadLabelInEnglish() {
+      OWLDataFactory df = ontologyManager.getOWLDataFactory();
+      OWLNamedIndividual phyloref =
+          df.getOWLNamedIndividual(IRI.create("http://example.org/phyloref1"));
+
+      // Look up labels in English.
+      Set<String> englishLabels = OWLHelper.getLabelsInEnglish(phyloref, testOntology);
+      assertEquals(1, englishLabels.size());
+      assertTrue(
+          englishLabels.contains("Label in English"),
+          "Label 'Label in English' correctly identified.");
+
+      // Look up Hindi or German names. Since Hindi is listed first, only the
+      // Hindi label is retrieved.
+      Set<String> hindiGermanLabels =
+          OWLHelper.getAnnotationLiteralsForEntity(
+              testOntology,
+              phyloref,
+              df.getOWLAnnotationProperty(OWLRDFVocabulary.RDFS_LABEL.getIRI()),
+              Arrays.asList("hi", "de"));
+      assertEquals(1, hindiGermanLabels.size());
+      assertTrue(
+          hindiGermanLabels.contains("अंग्रेजी में लेबल"),
+          "Label 'अंग्रेजी में लेबल' correctly identified.");
+
+      // Look up Spanish labels. Without a label, we'll default to the name without
+      // a language.
+      Set<String> spanishLabels =
+          OWLHelper.getAnnotationLiteralsForEntity(
+              testOntology,
+              phyloref,
+              df.getOWLAnnotationProperty(OWLRDFVocabulary.RDFS_LABEL.getIRI()),
+              Arrays.asList("es"));
+      assertEquals(1, spanishLabels.size());
+      assertTrue(
+          spanishLabels.contains("Label without a language"),
```

Comment:
You know, presumably you know what you're doing here, but to see a test that's trying to find a Spanish label, finds one without a language (?), and then treats its English-language content as success is a little bewildering.

Reply:
Sorry about the confusion! That should say "Label without a language tag", i.e. an `xsd:string` rather than an `rdfs:langString`. I've clarified across both OWLHelper and the tests how `getAnnotationLiteralsForEntity()` treats literals without language tags (312caf2). Is it better now, or would it make more sense to eliminate support for labels without language tags entirely?

Reply:
So you're still retrieving the label with no language tag in the result of querying spanish language labels. That's confusing – or expected behavior? If the latter (i.e., querying for labels in a language returns those tagged with that language and those untagged), then don't use `spanishLabels` to test for that, but query for English labels, or alternative say somewhere why the non-tagged result showing up here is expected (and where else it would be expected to show up).

Or maybe I'm just missing something.

Reply:
So the behavior right now is slightly different from your description:
1. If OWLHelper finds annotation literals with a requested language tag, it returns those literals.
2. If it doesn't find annotation literals with a requested language tag, but it does find annotation literals with no language tag, it returns those literals instead. This is primarily intended for the case where people forgot to add the correct language tags in the OWL input: it assumes that a label without language tags can be reasonably shown to people requesting any language.
3. If it only finds annotation literals with language tags not requested, then it returns the empty set.

So I think there's two problems here:
1. The test is confusing because it appears to be doing one thing (retrieving Spanish labels) when its actually doing something else (testing whether we fallback to using non-language-tagged labels correctly). I tried to clarify this in the comments (see below). I could clarify this by using a language code that [doesn't refer to an actual language](http://www.loc.gov/standards/iso639-2/faq.html#25), such as "zxx" (No linguistic content; Not applicable) or "und" (undetermined). 
https://github.com/phyloref/jphyloref/blob/312caf2dbae51141566a3405c02fcb4da97b4146/src/test/java/org/phyloref/jphyloref/helpers/OWLHelperTest.java#L114-L116

2. The documentation of the method isn't clear enough about what's going on here. I tried modifying it in this PR (see below), but I could modify it further.
https://github.com/phyloref/jphyloref/blob/312caf2dbae51141566a3405c02fcb4da97b4146/src/main/java/org/phyloref/jphyloref/helpers/OWLHelper.java#L55-L67

For the purposes of this PR, I think I should change this test so it clearly tests only the explicit retrieval of labels without language tags (c7ad134d9). I'll then open an issue to clarify the behavior I've described in the code and to write another test for that case. Alternatively, I could try clarifying the test or the documentation as described above. What do you think?

Reply:
It think a lot may be won already if you simply didn't name the variable `spanishLabels`. Because as you say, it's neither meant nor expected  to retrieve Spanish labels. Instead name it something more akin to what you're actually testing. E.g., `untaggedLabels`. That you expect to retrieve untagged labels by asking for labels with a language tag (say, `es`) that you're nearly certain aren't there can then be a simple comment.

Reply:
And it looks like that's pretty much what you did in 312caf2dbae51141566a3405c02fcb4da97b4146.

Reply:
I've split it into two different tests now: one (`labelsWithoutALanguageTag`) tests whether asking specifically for languages tagged "" returns annotations without a language tag (c7ad134), while the other (`labelsForUnknownLanguage`) tests whether `getAnnotationLiteralsForEntity()` will fall back to returning annotations without a language tag when no annotations have the requested language tag (005ac7e). Let me know if this makes sense!